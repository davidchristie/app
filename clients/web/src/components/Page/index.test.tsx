import { Page } from ".";
import { getByTestId, render, screen } from "../../testing";

describe("Page", () => {
  it("renders correctly", () => {
    const { container } = render(
      <Page>
        <div data-testid="children" />
      </Page>
    );
    const element = screen.getByTestId("Page");
    expect(getByTestId(element, "children")).toBeDefined();
    expect(element).toBe(container.firstChild);
    expect(element).toMatchSnapshot();
  });
});
