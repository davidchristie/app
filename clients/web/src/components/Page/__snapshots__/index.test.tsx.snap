// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders correctly 1`] = `
<div
  class="MuiBox-root MuiBox-root-1"
  data-testid="Page"
>
  <div
    data-testid="children"
  />
</div>
`;
